www/selectsource.php

```diff
@@ -160,6 +160,63 @@
     curl_close($ch);
 }
 
+// <timeout dialog>
+$timeoutDialogConfig = Configuration::getOptionalConfig('timeout_dialog.php');
```

Comment:
```suggestion
$timeoutDialogConfig = Configuration::getConfig('module_campusmultiauth.php')->getArray('timeout_dialog', []);
```
just an example, see comment